server/remote/event.go

```diff
@@ -26,6 +27,13 @@ type Event struct {
 	ResponseRequested          bool                 `json:"responseRequested,omitempty"`
 }
 
+func (e Event) GetMainID() string {
+	if e.RecurringEventID != "" {
+		return e.RecurringEventID
+	}
+	return e.ID
```

Comment:
Any reason to use `ID` over `ICalUID`? IIRC `ICalUID` is a better unique identifier since it's the same across all users' calendars, though there are different use cases for using one over the other

Reply:
I was pretty confused on why there were so many IDs for an event. But yes, using ICalUID should work in this scenario.